cypress/support/pages/home.pageObject.js

```diff
@@ -7,9 +7,16 @@ class HomePageObject extends PageObject {
     return cy.getByDataCy('username-link');
   }
 
-  assertHeaderContainUsername(username) {
-    this.usernameLink
-      .should('contain', username);
+  assertHeaderContainUsername = (username) => {
+    cy.get('[data-cy="username-link"]').should('contain', username);
+  };
+
+  clickNewArticle() {
+    cy.get('a.nav-link').contains('New Article').click();
+  }
+
+  clickSettings() {
```

Comment:
Same as above, find elements by data-qa attributes instead of text.

Reply:
I had wrote at Slack-chat about problems with data-qa attributes at my local machine
[https://mate-parttime.slack.com/archives/C02J1F3EUDB/p1691237811581409](url)
![e2e_11](https://github.com/mate-academy/qa_cypress_e2e_project/assets/13405550/9e15cae8-aa18-4b5e-923c-f5d701ccd5be)